packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Minor file changes to comply with eslint rules
+
+### Removed
+* Removed unnecessary disable eslint rules
```

Comment:
You don't need this 'removed block' thats mainly reserved for removing functionality

Reply:
Moved `Removed unnecessary disable eslint rules` under to under `Changed` instead of `Removed`: [7c3b572](https://github.com/cerner/terra-clinical/pull/669/commits/7c3b5725bed9eeffd8f4a7fc2ddbc2a738fb7774).